Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error: implicit declaration of function 'umem_startup' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2009-03-05 18:58:17.000000000 -0600
+++ configure	2020-12-31 14:44:57.000000000 -0600
@@ -4231,8 +4231,8 @@
   for (i = 0; i < 256; i++)
     if (XOR (islower (i), ISLOWER (i))
 	|| toupper (i) != TOUPPER (i))
-      exit(2);
-  exit (0);
+      return 2;
+  return 0;
 }
 _ACEOF
 rm -f conftest$ac_exeext
--- malloc.c.orig	2009-03-05 18:56:48.000000000 -0600
+++ malloc.c	2020-12-31 14:50:05.000000000 -0600
@@ -45,6 +45,8 @@
 # include <malloc.h>
 #endif
 
+extern void umem_startup(caddr_t, size_t, size_t, caddr_t, caddr_t);
+
 /*
  * malloc_data_t is an 8-byte structure which is located "before" the pointer
  * returned from {m,c,re}alloc and memalign.  The first four bytes give
--- umem_test.c.orig	2009-03-05 18:56:48.000000000 -0600
+++ umem_test.c	2020-12-31 14:53:16.000000000 -0600
@@ -4,6 +4,8 @@
 
 #include "umem.h"
 
+extern void umem_startup(caddr_t, size_t, size_t, caddr_t, caddr_t);
+
 int main(int argc, char *argv[])
 {
   char *foo;
--- umem_test2.c.orig	2009-03-05 18:56:48.000000000 -0600
+++ umem_test2.c	2020-12-31 14:57:13.000000000 -0600
@@ -13,6 +13,8 @@
 #define N_TESTSTRINGS (sizeof(TESTSTRINGS) / sizeof(TESTSTRINGS[0]))
 #define N_TESTS 1000
 
+extern void umem_startup(caddr_t, size_t, size_t, caddr_t, caddr_t);
+
 int
 main (int argc, char *argv[])
 {
